ocamlformat-help.txt

```diff
@@ -133,11 +133,12 @@ OPTIONS (CODE FORMATTING STYLE)
            Format code to fit within COLS columns. The default value is 80.
            Cannot be set in attributes.
 
-       --module-item-spacing={sparse|compact}
+       --module-item-spacing={sparse|preserve|compact}
            Spacing between items of structures and signatures. sparse will
-           always break a line between two items. compact will not leave open
-           lines between one-liners of similar sorts. The default value is
-           sparse.
+           always break a line between two items. preserve preserves the
+           spacing between items of structures and signatures. compact will
+           not leave open lines between one-liners of similar sorts. The
+           default value is sparse.
```

Comment:
I'm not sure if the 'preserve' terminology is quite right. The implementation of preserve is like compact, but will space things out if they are in the input. But it will add spaces between items as compact would. I expect that people reading 'preserve' will be surprised by that. I'm not thinking of an alternative term at the moment though. Perhaps it's enough to just expand the documentation.